src/main/java/seedu/contax/logic/parser/BatchCommandParser.java

```diff
@@ -27,13 +27,15 @@ public BatchCommand parse(String args) throws ParseException {
         String commandInput = argMultimap.getPreamble();
         String userValue = "";
         BatchType batchType = BatchType.EQUALS;
-        if (!argMultimap.arePrefixesPresent(PREFIX_SEARCH_TYPE) || argMultimap.getPreamble().isEmpty()
+        String searchType;
+        if (argMultimap.getValue(PREFIX_SEARCH_TYPE).isEmpty() || argMultimap.getPreamble().isEmpty()
             || (argMultimap.getValue(PREFIX_EQUALS).isEmpty()
                 && argMultimap.getValue(PREFIX_START_WITH).isEmpty()
                 && argMultimap.getValue(PREFIX_END_WITH).isEmpty())) {
-            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+            throw new ParseException(String.format(MESSAGE_INVALID_BATCH_INPUT,
```

Comment:
I dont think you should change this because it is standardized across all commands.

Reply:
thx change back